tests/telemetry_derived/clients_daily_v6/test_aggregation/main_summary_v4.schema.json

```diff
@@ -1003,5 +1003,39 @@
     "type": "STRING",
     "name": "isp_organization",
     "mode": "NULLABLE"
+  },
+  {
+    "fields": [
+      {
+        "type": "STRING",
+        "name": "key",
+       "mode": "NULLABLE"
+      },
+      {
+        "type": "INTEGER",
+        "name": "value",
+        "mode": "NULLABLE"
+      }
+    ],
+    "type": "RECORD",
+    "name": "scalar_parent_browser_search_ad_clicks",
+    "mode": "REPEATED"
+  },
+  {
+    "fields": [
+      {
+        "type": "STRING",
+        "name": "key",
+        "mode": "NULLABLE"
+      },
+      {
+        "type": "INTEGER",
+        "name": "value",
+        "mode": "NULLABLE"
+      }
+    ],
+    "type": "RECORD",
+    "name": "scalar_parent_browser_search_with_ads",
+    "mode": "REPEATED"
   }
-]
+]
```

Comment:
this should have a trailing newline

Reply:
@relud Thanks for your help in this pull request!